File: linux/ardium_source/presentation/04_Arrays/arrays.swift
// Topic: Arrays
// Sum elements of an array

import Foundation

let size = 5
var arr: [Int] = []

// Fill
for i in 0..<size {
    arr.append(i * 10)
}

// Sum
var sum = 0
for val in arr {
    sum += val
}

print("Sum: \(sum)")

File: linux/ardium_source/presentation/04_Arrays/arrays.rs
// Topic: Arrays (Vec)
// Sum elements of an array

fn main() {
    let size = 5;
    let mut arr = Vec::with_capacity(size);

    // Fill
    for i in 0..size {
        arr.push(i * 10);
    }

    // Sum
    let mut sum = 0;
    for val in &arr {
        sum += val;
    }

    println!("Sum: {}", sum);
}
